ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/synccommittee/SyncCommitteeSignatureValidator.java

```diff
@@ -107,6 +112,23 @@ public SyncCommitteeSignatureValidator(
             });
   }
 
+  boolean isSignatureForCurrentSlot(final UInt64 slot) {
+    if (recentChainData.getCurrentSlot().isEmpty()) {
+      return false;
+    }
+    final UInt64 slotMillis =
+        UInt64.valueOf(spec.atSlot(slot).getConfig().getSecondsPerSlot()).times(1000);
```

Comment:
probably worth having a `secondsToMillis` function or at least a constant for the 1000.  `AttestationValidator` already has a `secondsToMillis` private method for the same kind of thing so maybe that could go somewhere common and there's `TimeProvider.MILLIS_PER_SECOND` constant we could use. 

Although, given that `secondsPerSlot` is an int we could use:

```suggestion
        UInt64.valueOf(TimeUnit.SECONDS.toMillis(spec.atSlot(slot).getConfig().getSecondsPerSlot()));
```

Just gets awkward depending on when things become a UInt64.